### PR TITLE
Rollbar connection

### PR DESCRIPTION
**Pull request by azranel**
As title says, this adds Rollbar to project. It works only in `MIX_ENV=prod`.

### CODE REVIEW

**Comment by azranel** [comment]
@mwsteb Check dat shit, yo!

File: .travis.yml [hunk 1]
```diff
@@ -4,6 +4,6 @@ elixir:
 otp_release:
   - 18.2.1
 before_install:
-  - echo "use Mix.Config" >> ./config/dev.secret.exs
-  - echo "use Mix.Config" >> ./config/test.secret.exs
-  - echo "use Mix.Config" >> ./config/prod.secret.exs
+  - cp ./config/dev.secret.exs.sample >> ./config/dev.secret.exs
+  - cp ./config/test.secret.exs.sample >> ./config/test.secret.exs
+  - cp ./config/prod.secret.exs.sample >> ./config/prod.secret.exs
```
Comment (mwsteb) [inline .travis.yml]: `cp ./config/dev.secret.exs.sample >> ./config/dev.secret.exs` is not a thing :)
Try without appending: `cp ./config/dev.secret.exs.sample ./config/dev.secret.exs`
or just `cp ./config/dev.secret.exs{.sample,}`
I'm not sure the latter will work though.

Comment (azranel) [inline .travis.yml]: Heh, thanks for noticing 😄

**Comment by mwsteb** [comment]
LGTM

**Pull request merged**